src/map/MapRoutePath.js

```diff
@@ -79,17 +81,18 @@ const MapRoutePath = ({ name, positions, coordinates }) => {
     if (!coordinates) {
       coordinates = positions.map((item) => [item.longitude, item.latitude]);
     }
-    map.getSource(id)?.setData({
-      type: 'Feature',
-      geometry: {
-        type: 'LineString',
-        coordinates,
-      },
-      properties: {
-        name,
-        color: reportColor,
-      },
-    });
+    const maxSpeed = positions.map((p) => p.speed).reduce((a, b) => Math.max(a, b), -Infinity);
+    const features = [];
+    for (let i = 0; i < positions.length - 1; i += 1) {
```

Comment:
What happens if there's a single point?

Reply:
nothings is drawn

Reply:
But there should be one point, right?

Reply:
It will be in the other layer (the arrow)

Reply:
I see. That makes sense then.